schemas/jsonschema/FrameTransform.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$comment": "Generated from FrameTransform by @foxglove/message-schemas",
+  "title": "FrameTransform",
+  "description": "A transform between two named coordinate frames in 3D space",
```

Comment:
Not sure what the "named" part means / if it's necessary?

Reply:
Just that the parent and child are specified by name. I can remove it though.